fission-web/Main.hs

```diff
@@ -36,7 +36,7 @@ main = do
   Just manifest <- decodeFileStrict "./addon-manifest.json"
 
   _processCtx  <- mkDefaultProcessContext
-  _host        <- decode .!~ Web.Host "localhost:1337"
+  _host        <- decode .!~ Web.Host "https://runfission.com"
   _ipfsPath    <- decode .!~ IPFS.BinPath "/usr/local/bin/ipfs"
   _ipfsTimeout <- decode .!~ IPFS.Timeout 150
   _dbPath      <- decode .!~ DB.Path "web-api.sqlite"
```

Comment:
It's outside of the diff, so can't comment on the line, but line 54 looks broken to me 🤔 We should at least add an explicit default while we have the hood up.

```hs
isTLS <- getFlag "TLS" .!~ True
```

Reply:
Broken how? The flag works fine as is, but does default to False. I'll fix that

Reply:
> Broken how? 
This shouldn't typecheck: `if (isTLS :: Maybe Bool) then ... else ...`. Adding the `.!~` fixes it